app/layout.tsx

```diff
@@ -6,7 +6,27 @@ const inter = Inter({ subsets: ["latin"] });
 
 export const metadata: Metadata = {
   title: "tturl",
-  description: "shorten urls in one click",
+  description: "Shorten URLs in one click",
+  keywords: "URL shortener, shorten URLs, free URL shortener",
+  openGraph: {
+    title: "tturl",
+    description: "Shorten URLs in one click",
+    url: "https://tturl.netlify.app",
+    type: "website",
+    siteName: "tturl",
+    images: [
+      {
+        url: "https://tturl.netlify.app/og-image.jpg", 
```

Comment:
use the url from env file.. if i gonna get a domain.. its not necessary to change it in code right

Reply:
> use the url from env file.. if i gonna get a domain.. its not necessary to change it in code right

Could you please provide me url, I will change it